milestone1/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  app:
```

Comment:
Let's not call this `app`... We should split this and have server, client, and grid services (besides the SQL one already defined)